aikau/src/main/resources/alfresco/layout/VerticalReveal.js

```diff
@@ -18,35 +18,36 @@
  */
 
 /**
- * This is a BETA quality widget and not guaranteed for production use. 
+ * This is a BETA quality widget and not guaranteed for production use.
```

Comment:
We need better JSDoc here - I appreciate I should have done it the first time around, but we need to update the JSDoc on modules each time we edit them.